netbox/utilities/testing/views.py

```diff
@@ -583,6 +598,55 @@ def test_bulk_import_objects_with_permission(self):
             self.assertHttpStatus(self.client.post(self._get_url('import'), data), 200)
             self.assertEqual(self._get_queryset().count(), initial_count + len(self.csv_data) - 1)
 
+        @override_settings(EXEMPT_VIEW_PERMISSIONS=['*'])
+        def test_bulk_update_objects_with_permission(self):
+            if not self.csv_update_data:
+                return
+
+            data = {
+                'csv': self._get_csv_data(),
+            }
```

Comment:
This is defined below.
```suggestion
```

Reply:
it's different data - the first is self._get_csv_data, in the second instance it is the update_csv_data